core/roboconf-plugin-script/src/main/java/net/roboconf/plugin/script/internal/PluginScript.java

```diff
@@ -45,7 +46,7 @@
 import net.roboconf.plugin.api.template.InstanceTemplateHelper;
 
 /**
- * The plug-in invokes a shell script on every life cycle change.
+ * The plug-in invokes a (shell) script on every life cycle change.
```

Comment:
Given the way it works, we could run not only scripts but also any executable. I do not know if the "shell" mention is useful.